plenum/config.py

```diff
@@ -190,3 +190,5 @@
 # permissions for keyring dirs/files
 KEYRING_DIR_MODE = 0o700  # drwx------ 
 KEYRING_FILE_MODE = 0o600  # -rw-------
+
+ACCEPTED_DEVIATION_PREPREPARE_TIME = 10  # seconds
```

Comment:
In general, I think it's better to name constants in a way that includes their units, instead of documenting the units separately. So if the constant were named ACCEPTED_DEVIATION_PREPREPARE_SECS, the comment would be unnecessary, and you'd never have to wonder if people misunderstood the units.